src/Simplex/Chat.hs

```diff
@@ -7460,6 +7487,35 @@ closeFileHandle fileId files = do
   h_ <- atomically . stateTVar fs $ \m -> (M.lookup fileId m, M.delete fileId m)
   liftIO $ mapM_ hClose h_ `catchAll_` pure ()
 
+tagGroupChat' :: DB.Connection -> User -> GroupId -> ChatTagId -> ExceptT StoreError IO ChatResponse
+tagGroupChat' db user gId ctId = do
+  liftIO $ tagGroupChat db gId ctId
+  (allTags, chatTags) <- updatedGroupChatTags db user gId
+  pure $ CRChatTagged user allTags chatTags
+
+updatedGroupChatTags :: DB.Connection -> User -> GroupId -> ExceptT StoreError IO ([ChatTag], [ChatTag])
+updatedGroupChatTags db user gId = do
+  allTags <- liftIO $ getUserChatTags db user
+  let (groupTags, otherTags) = partition (\ChatTag {groupId} -> groupId == Just gId) allTags
+  pure (groupTags, otherTags)
```

Comment:
I don't understand names:
updatedGroupChatTags returns (groupTags, otherTags), but when it's called it's (allTags, chatTags)
chatTags I'd expect to be group tags, but it's in different place, and allTags I'd expect to be all tags, not just remaining tags..